src/submitter.py

```diff
@@ -486,6 +489,11 @@ def handler(event, context):
             payload_type = get_payload_type(raw_crash)
             payload_compressed = get_payload_compressed(raw_crash)
 
+            # Get the metadata.user_agent if there is one, or use default agent
+            user_agent = (
+                raw_crash.get("metadata", {}).get("user_agent") or DEFAULT_USER_AGENT
```

Comment:
instead of a fallback user agent should we set an empty one so that it's also empty at the destination?

Reply:
I thought about what to do here.

Currently, the socorro-submitter doesn't set a user agent at all, so the user agent defaults to `"python-requests/2.31.0"`. That gets picked up by the collector in stage and becomes the value of `metadata.user_agent` in the raw crash for all crash reports. That's not helpful.

When there is a user agent, we want the socorro-submitter to set the `User-Agent` header to the value that the prod collector captured. That's the easy case.

The harder case is what to do when there wasn't a `User-Agent` header and thus isn't a `metadata.user_agent` item in the raw crash First, I'm not sure that ever happens--I think all crash reports submitted to the prod collector get a user agent. I looked at 1,000 raw crashes and none of them are missing a user agent. I figured it may not matter what we do in the case where there isn't a `metadata.user_agent`. I think I'm going to leave it as is for now. It's not long for this world anyhow.